styles.scss

```diff
@@ -15,12 +15,14 @@ $bg-hover: #f1f5f9;
 html {
     font-family: 'Inter', sans-serif;
     font-size: 10px;
+    overflow-y: scroll;
```

Comment:
it's better to use `overflow-y: auto;`

Reply:
Helpful suggestion, thanks for pointing it out